nautobot_chatops/urls.py

```diff
@@ -33,14 +37,32 @@
     logger.warning("Grafana ChatOps integration is not available.", exc_info=True)
 
 urlpatterns = [
-    path("", NautobotHomeView.as_view(), name="home"),
+    path("", CommandLogListView.as_view(), name="commandlog_list"),
```

Comment:
Should this be?
```suggestion
    path("commandlog/", CommandLogListView.as_view(), name="commandlog_list"),
```

Reply:
Nope, this replaces that HomeView, kept in to keep the same feel from previous version.